anomalib/utils/callbacks/visualizer_callback.py

```diff
@@ -65,27 +68,30 @@ def _add_images(
         logger_type = type(module.logger).__name__.lower()
 
         # save image to respective logger
-        for log_to in module.hparams.project.log_images_to:
-            if log_to in loggers.AVAILABLE_LOGGERS:
+        for logger in self.loggers:
+            if logger in AVAILABLE_LOGGERS:
                 # check if logger object is same as the requested object
-                if log_to in logger_type and module.logger is not None:
+                if logger in logger_type and module.logger is not None:
                     module.logger.add_image(
                         image=visualizer.figure,
                         name=filename.parent.name + "_" + filename.name,
                         global_step=module.global_step,
                     )
                 else:
                     warn(
-                        f"Requested {log_to} logging but logger object is of type: {type(module.logger)}."
-                        f" Skipping logging to {log_to}"
+                        f"Requested {logger} logging but logger object is of type: {type(module.logger)}."
+                        f" Skipping logging to {logger}"
                     )
 
-        if "local" in module.hparams.project.log_images_to:
-            visualizer.save(Path(module.hparams.project.path) / "images" / filename.parent.name / filename.name)
+        if "local" in self.loggers:
+            if trainer.default_root_dir:
```

Comment:
I'm not hundred percent sure here. Should we choose `trainer.default_root_dir` to save/log our stuff or `trainer.log_dir`? What do you think? @ashwinvaidya17, @djdameln

Reply:
I guess we need a logger for accessing `log_dir`. If I am not wrong, we can save visualization results even without loggers. So in this case we should stick to `default_root_dir`.